docs/source/tools/non-repudiation.rst

```diff
@@ -0,0 +1,81 @@
+.. Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+.. SPDX-License-Identifier: Apache-2.0
+
+Non-repudiation
+###############
+
+The Daml Profiler is only available in the
```

Comment:
🍝 On it.

Reply:
Addressed by https://github.com/digital-asset/daml/pull/9288/commits/a7458331298a27b9c78f23bf4ba53692a83f1ae9